server/data/draftReportClientTypes.ts

```diff
@@ -15,3 +15,18 @@ export interface StaffDetails {
   missing?: boolean
   verified?: boolean
 }
+
+export type OffenderReport = {
+  date: moment.Moment
+  form: {
+    incidentDetails: Record<string, number>
+  }
```

Comment:
meh... 

could have potentially simplified this by doing a json query in the select to pull the thing you are interested to the top level

something a bit like:
```
select  r.incident_date date
          , r.form_response -> 'incidentDetails' ->> 'locationId' locationId
          , r.reporter_name reporter, r.status status
```

then your type could have been:
```
export type OffenderReport = {
  date: moment.Moment
  locationId: number
  reporter: string
  status: string
}
```

Reply:
will do this as a refactoring later.